src/edgepi/adc/adc_state.py

```diff
@@ -100,35 +100,44 @@ def __get_current_rtd_state(self) -> dict[str, PropertyValue]:
             "adc2_ref_inp": self.__get_state(ADCProperties.ADC2_REFMUX).code,
         }
 
-    def __get_rtd_state(self):
+    def __get_rtd_mode(self):
         """
-        Get on/off RTD state.
+        Get on/off RTD mode.
         Returns:
-            RTD
+            RTDModes (Enum): RTD_ON when RTD is on, RTD_OFF, when RTD is off, and None for undefined
+                             configuration.
         """
         # This is assuming, idac_1_mux - adc2_ref_inp are modified only when RTD is enabled.
         # compare the current rtd_state to RTDMode.RTD_OFF.value dictionary. If it doesn't match,
         # it means RTD is enabled
         rtd_state = self.__get_current_rtd_state()
         is_rtd_off = all(rtd_state.get(key)== value for key,value in RTDModes.RTD_OFF.value.items())
         if is_rtd_off:
-            return RTDModes.RTD_OFF.value
+            return RTDModes.RTD_OFF
 
         if all(rtd_state.get(key) == value for key, value in\
                (RTDModes.RTD_ON.value | ADC1RtdConfig.ON.value).items()):
-            return RTDModes.RTD_ON.value | ADC1RtdConfig.ON.value
+            return RTDModes.RTD_ON
 
         if all(rtd_state.get(key) == value for key, value in\
               (RTDModes.RTD_ON.value | ADC2RtdConfig.ON.value).items()):
-            return RTDModes.RTD_ON.value | ADC2RtdConfig.ON.value
+            return RTDModes.RTD_ON
```

Comment:
this code is repeated in `__get_rtd_adc_num`. just call that function.

Reply:
I tried to re-use as much as possible, but checking OFF state was inevitable.

[5b21157](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/296/commits/5b211574641b3380ae093f171ddd24ff5dc0bf94)